wfe/wfe.go

```diff
@@ -116,6 +116,15 @@ const (
 	defaultOrdersPerPage = 3
 )
 
+var goodSignatureAlgorithms = map[x509.SignatureAlgorithm]bool{
```

Comment:
If you care about saving a byte here or there, this can instead be a `map[x509.SignatureAlgorithm]struct{}`, which Go will optimize to having zero-byte values instead of one-byte values. But that's likely premature optimization here.

Reply:
If I was writing this from scratch I'd probably do that, but this list was taken verbatim from boulder.  Consistency seemed more important than a byte